doc/user/content/sql/functions/string_agg.md

```diff
@@ -11,23 +11,22 @@ input values (i.e. `value`) into [`text`](/sql/types/text). Each value after the
 first is preceded by its corresponding `delimiter`, where _null_ values are
 equivalent to an empty string.
 
+## Syntax
+
+{{< diagram "string-agg.svg" >}}
+
 ## Signatures
 
 Parameter | Type | Description
 ----------|------|------------
 _value_    | `text`  | The values to concatenate.
 _delimiter_  | `text`  | The values to precede the concatenated value.
 
+
 ### Return value
 
 `string_agg` returns a [`text`](/sql/types/text) value.
 
-Currently, this functions always executes on the data from `value` as if it were
```

Comment:
This comment is still correct! I think here is exactly where we should document ORDER BY as an argument to the aggregate function. It's still true that the outer ordering doesn't matter. And the way to address that is with an inner ordering. It probably makes sense to change this comment instead of outright remove it. Maybe

> If you need to perform aggregation in a specific order, you must specify `ORDER BY` within the aggregate function call itself. Otherwise incoming rows are not guaranteed any order.

If we do add a comment here, it should be replicated to the other 4 functions.